frontend/src/views/Project/RolePage/components/RolePermissionsSection/RolePermissionsSection.tsx

```diff
@@ -1,127 +1,52 @@
-import { useForm } from "react-hook-form";
+import { FormProvider, useForm } from "react-hook-form";
```

Comment:
hmmm for the default roles, I can add policies but it doesn't really get saved. I think in the previous version this was just disabled? 

<img width="1511" alt="image" src="https://github.com/user-attachments/assets/cb641b76-26ab-4a42-ad8c-4444b940f3cc">